src/SdkCommon/TestFramework/TestFramework.Tests/TestEnvironment/TestEnvironmentTests.cs

```diff
@@ -71,5 +71,21 @@ public void UpdateExistingEnvironmentUri(string connStr)
             Assert.Equal<string>("https://brazilus.management.azure.com/", SvcMgrUri);
             Assert.Equal<string>("https://portal.azure.com/", ibizaUri);
         }
+
+        [Theory(Skip = "Interactive Test")]
+        [InlineData("SubscriptionId=<subId>;RawToken=<rawToken.by2BGzn_-fzC4K-JWNRdG56MEBQtb1dGwxIeYY5jn_YgetUkkBR-jn8xjZQr_8-qAJ1ZwOlPdFAgvYsiw72Be7iBzo_9NTyJTUw4cGgSFI9Rtqx4IYGvJ_CcPpQWU4c1YFMqkUopU8I9eAOxtmcCpTx82Zq1uiWaVL62gdzBBC9I6WjOONSPoLVnK0LU5VmuZuS86efmdEpVhJ95llwAqAgoeFHeCx4ZJmj5y1ncOaSeQWHTfj5ovfPTMKGsWGWlbkbYDAzF0LBFo7Cau2wfkZf9nFmc9mcH05SPOnmQUwijpZSQ2CAXqK4f0EJnXI2ZenoNGfDYXQ>")]
```

Comment:
@markcowl yes all invalid tokens. This is just to show how the token looks like so that when this needs to be run, these values needs to be replaced with a valid token.